packages/plugins/graphql/server/services/builders/utils.js

```diff
@@ -95,16 +95,15 @@ module.exports = ({ strapi }) => {
      * Apply basic transform to GQL args
      */
     transformArgs(args, { contentType, usePagination = false } = {}) {
-      const { mappers } = getService('utils');
+      const { mappers, config } = getService('utils');
       const { pagination = {}, filters = {} } = args;
 
       // Init
       const newArgs = omit(['pagination', 'filters'], args);
 
       // Pagination
       if (usePagination) {
-        const defaultLimit = strapi.plugin('graphql').config('defaultLimit');
-        const maxLimit = strapi.plugin('graphql').config('maxLimit', -1);
+        const { defaultLimit, maxLimit } = config;
```

Comment:
no more defaults ?

Reply:
Handled in the new graphql config service